test/e2e/memcached_test.go

```diff
@@ -125,7 +130,22 @@ func TestMemcached(t *testing.T) {
 				}
 			}()
 		}
-		writeGoModReplace(t, sdkRepo, replace.repo, replace.ref)
+		modBytes, err := ioutil.ReadFile("go.mod")
+		if err != nil {
+			t.Fatalf("Failed to read go.mod: %v", err)
+		}
+		// Remove SDK repo dependency lines so we can parse the modfile.
+		sdkRe := regexp.MustCompile(`.*github\.com/operator-framework/operator-sdk.*`)
+		modBytes = sdkRe.ReplaceAll(modBytes, nil)
```

Comment:
You might be able to use `modFile.DropReplace` and `modFile.DropRequire` here, similar to my comment using the equivalent CLI flags in the ansible and helm scripts.

Reply:
Same as above.